test/integration/TestData.test.ts

```diff
@@ -338,6 +389,64 @@ describe('TestDataService', () => {
 
     });
 
+
+    test('Check generateActionMessages from ListingItem', async () => {
+        await testDataService.clean();
+
+        // get default profile
+        const defaultProfileModel = await profileService.getDefault();
+        const defaultProfile: resources.Profile = defaultProfileModel.toJSON();
+
+        const generateListingItemParams = new GenerateListingItemParams().toParamsArray();
+        const listingItems: Bookshelf.Collection<ListingItem> = await testDataService.generate<ListingItem>({
+            model: CreatableModel.LISTINGITEM,
+            amount: 1,
+            withRelated: true,
+            generateParams: generateListingItemParams
+        } as TestDataGenerateRequest);
+
+        const generateActionMessageParams = new GenerateActionMessageParams([
+            true,
+            true,
+            true,
+            ListingItemMessageType.MP_ITEM_ADD,
+            'nonce',
+            true,
+            listingItems[0].id,
+            defaultProfile.address,
+            1,
+            'actionmessage',
+            '0xf0afa0dbb1312410adaebccc12320567',
+            'dadadafgagag',
+            'testdatanotsorandommsgidfrom_generateListingItems'
+        ]).toParamsArray();
+
+        const actionMessages: Bookshelf.Collection<ActionMessage> = await testDataService.generate<ActionMessage>({
+            model: CreatableModel.ACTIONMESSAGE,
```

Comment:
all of the other generateXXX functions return resources.SomeResource[], so when using this, that's what I would expect this to always return too.

addXXX functions return Bookshelf models.